learning/index.md

```diff
@@ -30,6 +30,7 @@ The [Julia Language's YouTube channel](https://www.youtube.com/user/JuliaLanguag
 ## Text
 
 @@tight-list
+* [From zero to Julia!](https://techytok.com/from-zero-to-julia/) by [Aurelio Amerio](https://github.com/aurelio-amerio). An expanding series of short lessons about Julia, starting from beginner level and going up to deal with more advanced topics.
```

Comment:
What you've written seems ok, you could maybe go with "from introductory to more advanced topics"? Maybe also swap "lessons" for "tutorials"?